src/games/Lexible/models/PresenterModel.ts

```diff
@@ -690,6 +687,19 @@ export class LexiblePresenterModel extends ClusterfunPresenterModel<LexiblePlaye
         }
     }
 
+    //--------------------------------------------------------------------------------------
+    // 
+    //--------------------------------------------------------------------------------------
+    handleSwitchTeam(player: LexiblePlayer) {
+        const TeamA = this.players.filter(p => p.teamName === "A")
+        const TeamB = this.players.filter(p => p.teamName === "B")
+
+        if(player.teamName === "A" && TeamA.length < 2) return;
+        if(player.teamName === "B" && TeamB.length < 2) return;
+        player.teamName = player.teamName === "A" ? "B" : "A";
+
+        this.sendToPlayer(player.playerId, this.createPlayRequestMessage(player.teamName))
```

Comment:
Double-checking: when this Play Request message gets sent, the receiving player does not automatically start the game with an empty grid? The Play Request message does not contain the current presenter state, and in order for that to work, we need a separate delivery mechanism for the presenter state.

Reply:
Yes, this is a bit of an archaic interaction.  It used to be that play request was about starting a fresh round, but now it operates like an invite independent of game state.   It works for now, but might be worth a refactor for how the presenter communicates about game state.

Reply:
No worries, I'm using this message in much the same way in my new request/response system, though I've added a presenter state variable to it.